assets/stylesheets/giraffe-react.scss

```diff
@@ -108,7 +108,10 @@
         width: 300px;
     }
 }
-
+.contribute-fields {
+    margin-left: -5px;
```

Comment:
Have you checked this on steps 2 and 3?  They're also wrapped in `contribute-fields`.